allenact/algorithms/onpolicy_sync/engine.py

```diff
@@ -1736,7 +1734,8 @@ def run_eval(
                     lengths: List[int]
                     if self.num_active_samplers > 0:
                         lengths = self.vector_tasks.command(
-                            "sampler_attr", ["length"] * self.num_active_samplers,
+                            "sampler_attr",
```

Comment:
I see many formatting changes - are you also using `black` to ensure consistency?

Reply:
Yes, using Black, but must be using some different preferences with it (such as the number of characters in a line). I will try with `19.10b0` :)